components/places/src/import/fennec/bookmarks.rs

```diff
@@ -303,11 +303,11 @@ lazy_static::lazy_static! {
             isLocal
         )
         SELECT
-            normalize_root_guid(b.guid),
+            normalize_root_guid(sanitize_utf8(b.guid)),
             b.type,
-            (SELECT normalize_root_guid(p.guid) FROM fennec.bookmarks p WHERE p._id = b.parent),
+            IFNULL((SELECT normalize_root_guid(sanitize_utf8(p.guid)) FROM fennec.bookmarks p WHERE p._id = b.parent), 'unfiled_____'),
```

Comment:
GUIDs have stricter requirements than just being valid UTF-8 (Must be printable ASCII) -- THis will result in guids we can't sync to the server.

Reply:
> THis will result in guids we can't sync to the server.

That's handled by dogear. This will result in records we can at least read and hand off to dogear! IOW, any GUID touched by this function is already both (a) an invalid guid and (b) impossible for store.rs to read in the first place. This change just resolves (b), allowing our existing code to handle (a)